cypress/integration/loginWithRedirect.js

```diff
@@ -47,4 +49,46 @@ describe('loginWithRedirect', function() {
       shouldBe(pageParams.scope, 'openid profile email test test2');
     });
   });
+
+  it('can perform the login flow', () => {
+    whenReady().then(win => {
+      cy.loginNoCallback();
+
+      cy.url().then(url => shouldInclude(url, 'https://brucke.auth0.com'));
+
+      cy.get('#loaded').then(() => {
+        expect(win.sessionStorage.getItem('a0.spajs.txs')).to.exist;
+
+        cy.handleRedirectCallback().then(() => {
+          expect(win.sessionStorage.getItem('a0.spajs.txs')).to.not.exist;
+        });
+      });
+    });
+  });
+
+  it.only('can perform the login flow with cookie transactions', () => {
```

Comment:
I think this `only` is left here accidentally

Reply:
Whoops!